php_phongo.c

```diff
@@ -253,8 +253,8 @@ PHP_MINIT_FUNCTION(mongodb) /* {{{ */
 	php_phongo_bulkwrite_init_ce(INIT_FUNC_ARGS_PASSTHRU);
 	php_phongo_clientencryption_init_ce(INIT_FUNC_ARGS_PASSTHRU);
 	php_phongo_command_init_ce(INIT_FUNC_ARGS_PASSTHRU);
-	php_phongo_cursor_init_ce(INIT_FUNC_ARGS_PASSTHRU);
 	php_phongo_cursorid_init_ce(INIT_FUNC_ARGS_PASSTHRU);
+	php_phongo_cursor_init_ce(INIT_FUNC_ARGS_PASSTHRU);
```

Comment:
Why is this new position better?

Reply:
Without moving the class, PHP failed to initialise the extension as the cursor class was needed elsewhere.

Reply:
I meant to follow up on this. When testing this branch locally, I experienced no build error with the original order. I also don't see anything in `Cursor.c` that would depend on initialization of CursorId. There are references to `php_phongo_cursorid_ce`, but those are satisfied by the declarations in `phongo_classes.h`. The actual class entry doesn't need to be initialized until `getId()` is called.

Do you want to double-check this?

Reply:
Confirmed that the problem is indeed gone. I've reverted this change.